libsplinter/src/rest_api/mod.rs

```diff
@@ -957,6 +979,62 @@ impl RestApiBuilder {
     }
 }
 
+/// Retrieve the OpenId discovery document from the given link
+#[cfg(feature = "oauth-openid")]
+pub fn oauth_client_from_discovery_doc(
```

Comment:
This function and the struct should probably be under the auth::oauth module.

Reply:
Fixed